hack/after-build/update-swagger-spec.sh

```diff
@@ -66,12 +66,13 @@ APISERVER_PID=$!
 kube::util::wait_for_url "http://127.0.0.1:${API_PORT}/healthz" "apiserver: "
 
 SWAGGER_API_PATH="http://127.0.0.1:${API_PORT}/swaggerapi/"
-DEFAULT_GROUP_VERSIONS="v1 autoscaling/v1 batch/v1 extensions/v1beta1 apps/v1alpha1"
+DEFAULT_GROUP_VERSIONS="v1 autoscaling/v1 batch/v1 extensions/v1beta1 apps/v1alpha1 policy/v1alpha1"
```

Comment:
You won't be able to add your group here until you actually add a resource from it to the API. You need a change in master.go to add your disruption buget resource. Until then (like if you want to do this in two parts) just omit this change.